ha/halon/interface.c

```diff
@@ -770,6 +770,7 @@ static int halon_interface_level_enter(struct m0_module *module)
 	case M0_HALON_INTERFACE_LEVEL_SNS_CM_TRIGGER_FOPS:
 		m0_sns_cm_repair_trigger_fop_init();
 		m0_sns_cm_rebalance_trigger_fop_init();
+		m0_sns_cm_dtrebalance_trigger_fop_init();
```

Comment:
please add corresponding ...fop_fini() in halon_interface_level_leave() function

Reply:
Done.